dot/state/tries_test.go

```diff
@@ -27,18 +26,16 @@ func Test_NewTries(t *testing.T) {
 		rootToTrie: map[common.Hash]*trie.Trie{
 			tr.MustHash(): tr,
 		},
-		triesGauge: prometheus.NewGauge(prometheus.GaugeOpts{
-			Namespace: "gossamer_storage",
-			Name:      "tries_cached_total",
-			Help:      "total number of tries cached in memory",
-		}),
+		triesGauge:    triesGauge,
+		setCounter:    setCounter,
+		deleteCounter: deleteCounter,
 	}
 
 	assert.Equal(t, expectedTries, rootToTrie)
 }
 
 //go:generate mockgen -destination=mock_gauge_test.go -package $GOPACKAGE github.com/prometheus/client_golang/prometheus Gauge
-
+//go:generate mockgen -destination=mock_counter_test.go -package $GOPACKAGE github.com/prometheus/client_golang/prometheus Counter
```

Comment:
nit adding empty line below generate command

```suggestion
//go:generate mockgen -destination=mock_counter_test.go -package $GOPACKAGE github.com/prometheus/client_golang/prometheus Counter

```